infra/lambda/datasets-endpoint/conftest.py

```diff
@@ -0,0 +1,53 @@
+"""
+Pytest configuration file.
+"""
+
+import boto3
+import pytest
+
+DYNAMODB = boto3.resource("dynamodb")
+
+
+def pytest_configure():
+    """Share Dataset ID and title values between tests."""
+    return {"dataset_id": None, "dataset_type": None, "dataset_title": None, "owning_group": None}
+
+
+@pytest.fixture()
+def db_truncate(table_name="datasets"):
+    """Truncate DynamoDB table."""
+
+    table = DYNAMODB.Table(table_name)
+
+    # get the table keys
+    key_names = [key.get("AttributeName") for key in table.key_schema]
+
+    # only retrieve the keys for each item in the table (minimize data transfer)
+    projection_expression = ", ".join("#" + key for key in key_names)
+    expression_attribute_names = {"#" + key: key for key in key_names}
+
+    counter = 0
+    page = table.scan(
+        ProjectionExpression=projection_expression,
+        ExpressionAttributeNames=expression_attribute_names,
+    )
+
+    with table.batch_writer() as batch:
+        while page["Count"] > 0:
+            counter += page["Count"]
+
+            # delete items in batches
+            for itemKeys in page["Items"]:
+                batch.delete_item(Key=itemKeys)
```

Comment:
I don't think this is correct way to go. I saw those comments. But when you drop and re-create table in your tests you are not going to get the same configuration as when done with CDK, also re-creation of global indexes is not instant, so in our case it would be much slower.